packages/component-base/test/element-mixin.test.js

```diff
@@ -33,6 +34,15 @@ describe('ElementMixin', () => {
       expect(window.Vaadin.developmentModeCallback).to.be.instanceOf(Object);
       expect(window.Vaadin.developmentModeCallback['vaadin-usage-statistics']).to.be.instanceOf(Function);
     });
+
+    it('should set the Polymer cancelSyntheticClickEvents setting to false', async () => {
+      await import('@polymer/polymer/lib/utils/gestures.js');
+      const button = fixtureSync('<button></button>');
+      touchstart(button);
+      touchend(button);
+      const event = click(button);
+      expect(event.__polymerGesturesHandled).to.be.undefined;
```

Comment:
Checking for `event.__polymerGesturesHandled` doesn't seem meaningful from our point of view. Should be enough for us to check that `cancelSyntheticClickEvents` is `false` in `@polymer/polymer/lib/utils/settings.js`?

Reply:
Well, that's not too meaningful for us either actually :/

Reply:
Would be ideal if we could somehow reproduce an issue that importing Polymer's `gestures.js` causes...could be tricky to come up with

Reply:
Could this issue help somehow? At least the creator had a problem with it as well https://github.com/vaadin/web-components/issues/3300

Reply:
Understood, looks good enough as it is 👍

Reply:
Wait, actually, I can dynamically import the settings file and check the variable.

Reply:
@knoobie 

> Could this issue help somehow? At least the creator had a problem with it as well https://github.com/vaadin/web-components/issues/3300

We want to have our components work regardless of this setting. If a component doesn't work when this setting is disabled, I would say the component should be fixed.